.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "api-enumerations"]
+	path = api-enumerations
+	url = https://github.com/companieshouse/api-enumerations.git
+	branch = develop
```

Comment:
Doesn't api-enumerations use master as the default branch?

Reply:
thanks, updated